### DISCUSSION TITLE
Secret UI features?

### DISCUSSION THREAD

**Discussion by teymour-aldridge** [servo/servo#27979] [Q&A] [answered]
I think I've seen photos of people bringing up a search bar and tabs in Servo, which I think were put there for convenience while debugging/testing.

Does anyone know how to enable them?

**Comment by jdm** [comment, marked as answer]
The only secret UI is ctr+L (or cmd+L on macOS) to bring up a janky URL input. There is no UI support for tabs yet.

**Reply by teymour-aldridge** [reply]
Thanks! Are tabs planned (probably a huge amount of work)?

**Reply by jdm** [reply]
Generally yes, but without a concrete plan right now. https://github.com/servo/servo/issues/20335 is probably the best source of information right now.

**Comment by follower** [comment]
In case it's helpful, if anyone is looking for the "non-secret" UI keyboard shortcuts, they're here: https://github.com/servo/servo/blob/74f1eb199e67e82ff713f77f7803f02a25966628/README.md#keyboard-shortcuts
